cni.go

```diff
@@ -45,6 +49,8 @@ type CNI interface {
 	Status() error
 	// GetConfig returns a copy of the CNI plugin configurations as parsed by CNI
 	GetConfig() *ConfigResult
+	//BuildCNIMultiNetwork returns a list of networks that match the network name in the cni cache
+	BuildCNIMultiNetwork(networkNames []*NetworkInterface) []*Network
```

Comment:
```suggestion
	BuildMultiNetwork(networkNames []*NetworkInterface) []*Network
```

Reply:
Good check, let me move this back to draft. I was a little overexcited to get this PR up!

Reply:
I need to update the comment above as well